src/modules/FileSystemUtility.js

```diff
@@ -251,54 +259,70 @@ module.exports.exportSession = async (params) => {
         .then((data) => {
           const pdfName = "yattie-session-" + timestamp + "-report.pdf";
           const pdfPath = path.join(configDir, "sessions", id, pdfName);
-          fs.writeFile(pdfPath, data, (error) => {
-            if (error) {
-              return Promise.resolve({
-                status: STATUSES.ERROR,
-                message: error,
-              });
-            }
-            try {
-              const zip = new AdmZip();
-
-              const items = persistenceUtility.getItems();
-              items.map((item) => {
-                if (item.filePath) {
-                  const sanitizedPath =
-                    item.filePath.substring(item.filePath.length - 1) !== "?"
-                      ? item.filePath
-                      : item.filePath.substring(0, item.filePath.length - 1);
-                  zip.addLocalFile(sanitizedPath, FILE_TYPES[item.fileType]);
+          params.type === "pdf"
```

Comment:
@dockabernathy01 please put this in an `if` statement instead.  It is hard to read the multi-line ternary operator.